src/theme.js

```diff
@@ -41,55 +43,79 @@ export default {
-      fontSize: "16px",
-      lineHeight: "22px",
-    },
-    body2: {
-      color: color.deepPurple,
-      fontFamily: "Open Sans, Ariomo, Arial, sans-serif",
-      fontSize: "14px",
-      lineHeight: "22px",
-    },
     h1: {
-      fontFamily: "Ariomo, Arial, sans-serif",
+      fontFamily: "Open Sans, sans-serif",
       fontWeight: "bold",
-      fontSize: "36px",
-      lineHeight: "40px",
-      color: color.deepPurple,
+      fontSize: "32px",
+      lineHeight: "48px",
+      color: color.blue,
     },
     h2: {
-      fontFamily: "Ariomo, Arial, sans-serif",
-      fontWeight: "bold",
-      fontSize: "22px",
-      lineHeight: "24px",
-      color: color.deepPurple,
+      fontFamily: "Open Sans, sans-serif",
+      fontWeight: "semi-bold",
+      fontSize: "24px",
+      lineHeight: "36px",
+      color: color.blue,
     },
     h3: {
-      fontFamily: "Open Sans, Arial, sans-serif",
-      fontSize: "24px",
-      fontWeight: 600,
-      lineHeight: "32px",
-      textTransform: "capitalize",
-      color: color.deepPurple,
+      fontFamily: "Open Sans, sans-serif",
+      fontWeight: "semi-bold",
+      fontSize: "20px",
+      lineHeight: "30px",
+      color: color.blue,
     },
     h4: {
-      fontFamily: "Ariomo, Arial, sans-serif",
-      fontWeight: "bold",
-      fontSize: "14px",
-      lineHeight: "16px",
-      textTransform: "uppercase",
-      letterSpacing: "0.01em",
-      color: color.deepPurple,
+      fontFamily: "Open Sans, sans-serif",
+      fontWeight: "semi-bold",
+      fontSize: "16px",
+      lineHeight: "24px",
+      color: color.blue,
     },
     h5: {
-      fontFamily: "Ariomo, Arial, sans-serif",
+      // still available as some components/pages depend on this
+      fontFamily: "Open Sans, sans-serif",
+      fontWeight: "semi-bold",
+      fontSize: "16px",
+      lineHeight: "24px",
+      color: color.blue,
+    },
+    body1: {
+      fontFamily: "Open Sans, sans-serif",
+      fontWeight: "regular",
+      fontSize: "16px",
+      lineHeight: "24px",
+      color: color.black,
+    },
+    body2: {
+      // same as for h5
+      fontFamily: "Open Sans, sans-serif",
+      fontWeight: "regular",
+      fontSize: "16px",
+      lineHeight: "24px",
+      color: color.black,
+    },
+    button: {
+      fontFamily: "Open Sans, sans-serif",
       fontWeight: "bold",
       fontSize: "16px",
-      lineHeight: "20px",
-      color: color.deepPurple,
+      lineHeight: "24px",
+      textTransfrom: "uppercase",
+      color: color.black,
+    },
+    caption: {
+      fontFamily: "Open Sans, sans-serif",
+      fontWeight: "regular",
+      fontSize: "12px",
+      lineHeight: "18px",
+      color: color.black,
+    },
+    overline: {
+      fontFamily: "Open Sans, sans-serif",
+      fontWeight: "regular",
+      fontSize: "12px",
+      lineHeight: "18px",
+      textTransfrom: "uppercase",
+      color: color.black,
     },
```

Comment:
Comment applies to entire file. Same as above, are font's part of this tickets scope?